src/panels/config/automation/action/types/ha-automation-action-choose.ts

```diff
@@ -208,6 +304,9 @@ export class HaChooseAction extends LitElement implements ActionElement {
         .link-button-row {
           padding: 14px;
         }
+        .card-content {
+          padding: 0 0 8px 8px;
```

Comment:
```suggestion
          padding: 0 16px 16px 16px;
```
To have consistent padding with other blocks

Reply:
We can also remove the `padding: 4px;` to the `ha-icon-button`

Reply:
I did this, but also removed the bottom padding from the bottom of link-button-row. 

It was having 14px bottom padding, which combined up with the 16px card-content padding, and having 30px of bottom padding didn't look consistent. 

Just to be clear, you're not looking for any further changes at this time related to the description/aliases?